tests/test_interface.rb

```diff
@@ -137,20 +137,27 @@ def interface_count
     arr.count
   end
 
+  def test_interface_capabilities
+    if validate_property_excluded?('interface', 'capabilities')
+      assert_empty(Interface.capabilities(interfaces[0]))
+    else
+      refute_nil(Interface.capabilities(interfaces[0]),
+                 'Interface.capabilities should not return an empty hash')
```

Comment:
Please add asserts for getting the `Model` value from both `raw` and `hash` modes.